.github/workflows/_test.yml

```diff
@@ -21,7 +21,11 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
-    name: "test #${{ matrix.python-version }}"
+        core-version:
+          - ">=0.2.0,<0.3.0"
+          - "latest"
```

Comment:
this won't test with core 0.3 until it's out of pre release I think

Reply:
yea, it needs to be flipped, will update